db/db_impl.cc

```diff
@@ -348,9 +348,12 @@ Status DBImpl::CloseHelper() {
   flush_scheduler_.Clear();
 
   while (!flush_queue_.empty()) {
-    auto cfd = PopFirstFromFlushQueue();
-    if (cfd->Unref()) {
-      delete cfd;
+    auto flush_req = PopFirstFromFlushQueue();
```

Comment:
Can you spell the type out? Based on Google C++ Style, it is recommended to spell the type if it's not obvious from the context: https://google.github.io/styleguide/cppguide.html#auto

Reply:
Done.